src/algebra/lattice_ordered_group.lean

```diff
@@ -550,4 +550,38 @@ begin
     apply lattice_ordered_comm_group.inv_le_abs, }
 end
 
+/--
+Let `α` be a lattice ordered commutative group and let `a` and `b` be elements in `α`. Then,
+`|a - b|` the absolute value of `a - b`, is equal to `|b - a|`, the absolute value of `b - a`.
+-/
+@[to_additive]
+lemma abs_inv_invariant (a b : α) : |a/b| = |b/a| :=
+begin
+  unfold has_abs.abs,
+  rw [inv_div' a b, ← inv_inv (a / b), inv_div', sup_comm],
+end
+
+/--
+Let `α` be a lattice ordered commutative group and let `a` and `b` be elements in `α`. Let `|a|`,
+`|b|`, `||a| - |b||` and `|a - b|` be the absolute values of `a`, `b`, `|a| - |b|` and `a - b`. Then
+ `||a| - |b||` is dominated by `|a - b|`.
+-/
+@[to_additive]
```

Comment:
At a minimum this should be
```suggestion
@[to_additive /-"
Let `α` be a lattice ordered commutative group and let `a` and `b` be elements in `α`. Let `|a|`,
`|b|`, `||a| - |b||` and `|a - b|` be the absolute values of `a`, `b`, `|a| - |b|` and `a - b`. Then
 `||a| - |b||` is dominated by `|a - b|`.
"-/]
```
else the docstring is incorrectly attached to the `/` version.

I'd lean towards just removing this docstring though, it's a lot of words to describe something that should be obvious from reading:
```suggestion
@[to_additive]
```
Let's see what @sgouezel thinks of this comment.

Reply:
I've moved all of the docstrings into the `to_additive` - I thought it more useful to document the additive case as:
a) In practice I suspect that's what most people will make use of;
b) This way you get to see both the additive and multiplicative forms of the equations when scanning through the file (I imagine more eyes will be looking for |a + b| ≤ |a| + |b| than |a * b| ≤ |a| * |b|).

My D.Phil supervisor trained me to define all terms precisely in statements, and I find this greatly improves readability, particularly when returning to work some years later!